CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [2.2.3] - 2022-06-07
 
 ### Changed
+- Updated conjur-env dependencies to latest versions (github.com/cyberark/summon -> v0.9.2,
```

Comment:
We just cut the 2.2.3 release - should this be in either the Unreleased or a new 2.2.4 section?

Reply:
Yes it should, and I could have sworn I checked that. Will update. Thanks!